src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -314,6 +317,7 @@ private void TrackIndexMetrics(NuGetSearcherManager searcherManager, SearchTelem
             catch (Exception e)
             {
                 await _responseWriter.WriteResponseAsync(context, e, _logger);
+                throw e;
```

Comment:
There is ClientException catch above this block, when are these thrown? Do we want to log it?